pkg/options/options.go

```diff
@@ -148,6 +148,17 @@ type UnityAndroid struct {
 	VersionCode   string      `help:"The version code of this build of the application"`
 	VersionName   string      `help:"The version of the application"`
 }
+type Breakpad struct {
+	SymFilePath     utils.Paths `arg:"" name:"path" help:"The path to the Unity symbols (.zip) file to upload (or directory containing it)" type:"path"`
+	ProjectRoot     string      `help:"The path to strip from the beginning of source file names referenced in stacktraces on the BugSnag dashboard" type:"path"`
+	VersionCode     string      `help:"The version code of this build of the application"`
+	VersionName     string      `help:"The version of the application"`
+	OsName          string      `help:"The name of the operating system that generated the breakpad file"`
+	CpuInfo         string      `help:"The CPU architecture of the system that generated the breakpad file"`
+	DebugFile       string      `help:"The path to the code file that the breakpad file relates to" type:"path"`
+	DebugIdentifier string      `help:"The debug identifier of the code file that the breakpad file relates to"`
+	Product         string      `help:"The product of the code file that the breakpad file relates to"`
```

Comment:
Some changes to match the API docs:

```suggestion
	Path            utils.Paths `arg:"" name:"path" help:"The path to the symbol files (.sym) to upload (or directory containing them)" type:"path"`
	CpuArch         string      `help:"The CPU architecture that the module was built for"`
	CodeFile        string      `help:"The basename of the module"`
	DebugFile       string      `help:"The basename of the debug file"`
	DebugIdentifier string      `help:"The debug file's identifier"`
	ProductName     string      `help:"The product name"`
	ProjectRoot     string      `help:"The path to strip from the beginning of source file names referenced in stacktraces on the BugSnag dashboard" type:"path"`
	OsName          string      `help:"The name of the operating system that the module was built for"`
	VersionName     string      `help:"The version of the application"`
```

- Rename SymFilePath -> Path (for consistency)
- Rename CpuInfo -> CpuArch
- Rename Product -> ProductName
- Remove VersionCode (not in API)
- Reordered alphabetically
- Used descriptions from docs.